app/code/Magento/AdminNotification/Test/Integration/Controller/Adminhtml/Notification/MassMarkAsReadTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+/***
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\AdminNotification\Test\Integration\Controller\Adminhtml\Notification;
+
+use Magento\Framework\App\Request\Http as HttpRequest;
+use Magento\Framework\Exception\AuthenticationException;
+use Magento\TestFramework\TestCase\AbstractBackendController;
+
+class MassMarkAsReadTest extends AbstractBackendController
+{
+    /**
+     * @return void
+     * @throws AuthenticationException
+     */
+    public function setUp(): void
```

Comment:
Please revert it, it should be protected

```suggestion
    protected function setUp(): void
```